test-suite-kotlin-ksp/src/test/kotlin/io/micronaut/docs/aop/scheduled/ScheduledExample.kt

```diff
@@ -15,10 +15,12 @@
  */
 package io.micronaut.docs.aop.scheduled
 
+import io.micronaut.context.annotation.Requires
 import io.micronaut.scheduling.annotation.Scheduled
 
 import jakarta.inject.Singleton
 
+@Requires(property = "spec.name", value = "ScheduledExampleTest")
```

Comment:
otherwise I was getting

```
Caused by: io.micronaut.context.exceptions.NoSuchBeanException: No bean of type [java.util.concurrent.ExecutorService] exists for the given qualifier: @Named('scheduled'). Make sure the bean is not disabled by bean requirements (enable trace logging for 'io.micronaut.context.condition' to check) and if the bean is enabled then ensure the class is declared a bean and annotation processing is enabled (for Java and Kotlin the 'micronaut-inject-java' dependency should be configured as an annotation processor).
```

There does not seem to be a test which uses this class. It is only used for the documentation examples.